app/models/reservation.rb

```diff
@@ -368,8 +368,9 @@ def billable_duration_mins
     when InstrumentPricePolicy::CHARGE_FOR.fetch(:usage)
       TimeRange.new(actual_start_at, actual_end_at).duration_mins
     when InstrumentPricePolicy::CHARGE_FOR.fetch(:overage)
+      start_time = [reserve_start_at, actual_start_at].min
       end_time = [reserve_end_at, actual_end_at].max
```

Comment:
Have you confirmed the cause of this?
I think the problem is raised on this line - according to [rollbar](https://app.rollbar.com/a/corum/fix/item/nucore-umass/66).  In my local testing, `TimeRange` seems to handle `nil`s just fine:
```
[3]nucore-open(main)> TimeRange.new(Time.current, nil).duration_mins
nil
[4]nucore-open(main)> TimeRange.new(nil, Time.current).duration_mins
nil
```
... while `max` raises the error seen in rollbar:
```
[2]nucore-open(main)> [Time.current, nil].max
ArgumentError: comparison of NilClass with ActiveSupport::TimeWithZone failed
```